src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -249,17 +249,20 @@ public async Task<ActionResult> Setup(SetupApiViewModel model)
             var setupContext = new SetupContext
             {
                 ShellSettings = shellSettings,
-                SiteName = model.SiteName,
                 EnabledFeatures = null, // default list,
-                AdminUsername = model.UserName,
-                AdminEmail = model.Email,
-                AdminPassword = model.Password,
                 Errors = new Dictionary<string, string>(),
                 Recipe = recipeDescriptor,
-                SiteTimeZone = model.SiteTimeZone,
-                DatabaseProvider = selectedProvider.Value,
-                DatabaseConnectionString = connectionString,
-                DatabaseTablePrefix = tablePrefix
+                Properties = new Dictionary<string, object>()
+                {
+                    {"SiteName", model.SiteName},
```

Comment:
Format, make a space before and after the brackets

Reply:
Can you fix the formatting ? Did'nt quite get it.

Reply:
Add a space after / before eah bracket `{ "SiteName", model.SiteName }`

Reply:
Under Visual Studio, you can fomat your file by doing ctrl+K+D

Reply:
Exactly!! CRTL+K+D is my favorite ;)

Reply:
formatting done.  Im using Rider as IDE so it has diffrent opinions for formatting.